client/src/components/AllUsers.js

```diff
@@ -33,26 +33,33 @@ const AllUsers = () => {
     setSearch(value);
   };
 
-  function handleDailyFilter() {
-    setFilter("daily");
-  }
-
-  function handleWeeklyFilter() {
-    setFilter("weekly");
-  }
-
-  function handleMonthlyFilter() {
-    setFilter("monthly");
-  }
+  const handleFilter = (event) => {
+    event.preventDefault();
+    if (event.target.innerText === "Daily") {
+      setFilter("daily");
+      setSelected("daily");
+    } else if (event.target.innerText === "Weekly") {
+      setFilter("weekly");
+      setSelected("weekly");
+    } else if (event.target.innerText === "Monthly") {
+      setFilter("monthly");
+      setSelected("monthly");
+    }
+  };
 
   return (
     <div>
       <div>
         <NavBar />
       </div>
-      {trainees.length <= 1 ? (
+      {trainees.length < 1 ? (
```

Comment:
I think the logic here is incorrect - should be > 1 so that the filters are displayed when there are results?
<img width="804" alt="image" src="https://user-images.githubusercontent.com/38047481/224979510-0501d48f-305f-40fe-aa8a-049075e3f726.png">